tilequeue/rawr.py

```diff
@@ -120,7 +120,7 @@ def __call__(self, coords):
 
         if self.logger:
             self.logger.info(
-                'Expiry processed: '
+                'Rawr tiles enqueued: '
```

Comment:
Well spotted! :smile:

Reply:
It was correct at the time, in that it was only enqueueing tile from expiry lists, but now there are multiple cases :)